include/odp/api/spec/timer_types.h

```diff
@@ -548,36 +531,6 @@ typedef enum {
 
 } odp_timer_retval_t;
 
-/**
- * For backwards compatibility, odp_timer_set_t is synonym of odp_timer_retval_t
- *
- * @deprecated Use odp_timer_retval_t instead.
- */
-typedef odp_timer_retval_t odp_timer_set_t;
```

Comment:
The deprecation comment was added in June (v1.41.1) and no deprecation macros have been used. Safe to be removed?

Reply:
Added the typedef back in V2. Will deprecate it in a separate PR.